spec/amp-actions-and-events.md

```diff
@@ -510,28 +515,36 @@ event.response</pre></td>
   </tr>
 </table>
 
-### amp-selector
+### amp-selector <a name="amp-selector-1"></a>
 
 <table>
   <tr>
     <th>Action</th>
     <th>Description</th>
   </tr>
+  <tr>
+    <td><code>clear</code></td>
+    <td>Clears all selections from a defined <code>amp-selector</code>.</td>
+  </tr>
   <tr>
     <td><code>selectUp(delta=INTEGER)</code></td>
-    <td>Moves the selection up by the value of `delta`. The default `delta` is set to 1.</td>
+    <td>Moves the selection up by the value of `delta`. The default `delta` is set to -1.</td>
```

Comment:
Add: "If no options are selected, the selected state will become the value of the last option."